### PR TITLE
[MCHANGES-429] Remove ClassicJiraDownloader

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: src/main/java/org/apache/maven/plugins/jira/JiraReport.java [hunk 1]
```diff
@@ -333,9 +333,7 @@ public void executeReport(Locale locale) throws MavenReportException {
             if (mockDownloader != null) {
                 issueDownloader = mockDownloader;
             } else {
-                AdaptiveJiraDownloader downloader = new AdaptiveJiraDownloader();
-                downloader.setForceClassic(forceRss);
```
Comment (slawekjaranowski) [inline src/main/java/org/apache/maven/plugins/jira/JiraReport.java:337]: `forceRss` parameter can also be removed

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]
Please change subject of PR and commits .... should be  - like 
`Remove ClassicJiraDownloader`

File: src/main/java/org/apache/maven/plugins/jira/JiraReport.java (hunk 1 above)
Comment (elharo) [reply src/main/java/org/apache/maven/plugins/jira/JiraReport.java:337]: done

**Review by slawekjaranowski — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #307

**Pull request merged**